### PR TITLE
Add multi tenant guide

### PR DESCRIPTION
**Pull request by mooreds**
This guide helps folks through multi tenant setup.

### CODE REVIEW

**Review by robotdan — COMMENTED** [review]
Looks good. 

I had a bit of trouble wrapping my head around the control plane and the data plane concepts well enough to context switch while reading the document to understand which plane was being described. 

Since this is sort of a cookbook for setting up a multi-tenant SaaS and leveraging FusionAuth, this almost feels like an expert advice. Or if we leave it here, perhaps we can either adjust the tittle, or description at the top of the article. From reading through it, it felt more like, here is "How to build a multi-tenant SaaS using FusionAuth" - then "How to set up multi-tenancy in FusionAuth".

Just my opinion.

File: site/_includes/_example_apps.liquid [hunk 1]
```diff
@@ -13,7 +13,7 @@
       {% for app in sorteditems %}
         <li>
           <a href="{{ app.url }}">
-            {{ app.name | capitalize }}
+            {{ app.name }}
```
[the remark this answers is not in the document]
Comment (mooreds) [reply site/_includes/_example_apps.liquid:16]: WFM

File: site/docs/v1/tech/guides/multi-tenant.adoc [hunk 2]
```diff
@@ -0,0 +1,772 @@
...
+=== Defining The Tenant Object
+
+The tenant object exists in your application. As outlined in <<The Tenant Object>>, this entity stores all tenant related information, whether for FusionAuth or application specific. Here's a sample `tenant` table definition:
+
+[source,sql,title=Tenant object creation DDL]
+----
+CREATE TABLE `tenant` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `user_id` int NOT NULL,
+  `hostname` varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
```
[the remark this answers is not in the document]
Comment (mooreds) [reply site/docs/v1/tech/guides/multi-tenant.adoc:261]: Good catch!

File: site/docs/v1/tech/guides/multi-tenant.adoc [hunk 3]
```diff
@@ -0,0 +1,772 @@
...
+
+The creator of a PPVC tenant will be able to pick a hostname and a display background color. If someone from Hooli joins, they could set up `hooli.piedpipervideochat.com`, for example. As mentioned above, Slack uses hostnames to differentiate tenants, and PPVC will as well.
+
+== Components of a Multi-Tenant Setup
+
+While FusionAuth multi-tenancy can help with many different scenarios, as mentioned above, this rest of this guide will focus on building out the Pied Piper Video Chat SaaS application.
+
+Unapologetically borrowing from https://en.wikipedia.org/wiki/Control_plane[network routing], let's use the concept of a control plane application and data plane applications. A control plane application manages user and tenant creation. This application is where you can integrate billing and general account management if charging for your application. Then there's the application that users will access to actually video chat. These will be called data plane applications.
```
[the remark this answers is not in the document]
Comment (mooreds) [reply site/docs/v1/tech/guides/multi-tenant.adoc]: Nope, only one control plane application :)

File: site/docs/v1/tech/guides/multi-tenant.adoc [hunk 4]
```diff
@@ -0,0 +1,772 @@
...
+{% plantuml source: _diagrams/docs/guides/data-plane-login.plantuml, alt: "Login process for the data plane application." %}
+++++
+
+The users of data plane app can register or log in and see the chat window (or at least a note about the chat functionality that will later be built out). 
+
+image::guides/multi-tenant/data-plane-logged-in-view.png[The data plane application interface.,width=1200,role=bottom-cropped]
+
+Each of the data plane applications are in a different FusionAuth tenant. Different users can have the same email address, as you can see in the below screenshot, where the `jian@fusionauth.io` user exists in the default tenant and the `ppvctest2` tenant:
+
+image::guides/multi-tenant/users-dashboard-fusionauth.png[FusionAuth user dashboard.,width=1200,role=bottom-cropped]
```
[the remark this answers is not in the document]
Comment (mooreds) [reply site/docs/v1/tech/guides/multi-tenant.adoc:223]: Nope, I was resizing and the I cropped.

[the remark this answers is not in the document]
Comment (mooreds) [reply site/docs/v1/tech/guides/multi-tenant.adoc:223]: I think the issue is that I had the browser at different magnification levels.

**Pull request merged**